Content.Client/Chat/UI/EmotesMenu.xaml

```diff
@@ -1,4 +1,4 @@
-﻿<ui:RadialMenu xmlns="https://spacestation14.io"
+<ui:RadialMenu xmlns="https://spacestation14.io"
```

Comment:
what the hell changed? i cannot see even whitespace changes! WTF

Reply:
UTF8 bom moment, don't worry about it